doc/build-unix.md

```diff
@@ -73,11 +73,13 @@ GUI dependencies:
 Bitcoin Core includes a GUI built with the cross-platform Qt Framework. To compile the GUI, we need to install
 the necessary parts of Qt, the libqrencode and pass `-DBUILD_GUI=ON`. Skip if you don't intend to use the GUI.
 
-    sudo apt-get install qtbase5-dev qttools5-dev qttools5-dev-tools
+    sudo apt-get install qt6-base-dev qt6-tools-dev qt6-l10n-tools
+
+For Qt 6.5 and later, the `libxcb-cursor0` package must be installed at runtime.
```

Comment:
Still not sure I understand this addtion. The people reading these docs (self compiling) don't need it, because all the deps they need should be in the docs already, and the person using the release binaries isn't reading the self-compilation docs, so isn't going to see it?

Reply:
What would be a better way to proceed here?